packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -754,8 +762,6 @@ const DashboardEditor = ({
                         isResizable={isCardResizable}
                         i18n={mergedI18n}
                         isSelected={isSelected}
-                        getValidDataItems={getValidDataItems}
-                        dataItems={dataItems}
```

Comment:
This doesn't seem to be a backwards compatible change. Shouldn't these be left and deprecated with a warning in PropTypes?

Reply:
hmm, I'll check the external props interface, I didn't think I changed anything there (at least I didn't intend to).  The DashboardEditor @kevinsperrine

Reply:
Got it. So they were being passed to the `BarChartCard` via the `EditorBarChartCard` (https://github.com/carbon-design-system/carbon-addons-iot-react/pull/2790/files#diff-2ff08df7c609521e34782529133308227cf907b56069ea653c8b6ef6eeafed77L328) but the BarChartCard doesn't actually use them. They are only used by the Form side of the DashboardEditor, not anywhere in the Renderer... 👍